flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/InternalSinkWriterMetricGroup.java

```diff
@@ -55,6 +55,12 @@ public static InternalSinkWriterMetricGroup mock(MetricGroup metricGroup) {
                 metricGroup, UnregisteredMetricsGroup.createOperatorIOMetricGroup());
```

Comment:
If we use `UnregisteredMetricGroups.createUnregisteredOperatorMetricGroup().getIOMetricGroup()` here, maybe we can get rid of your new `mock`? Then we could just use `metricGroup#getIOMetricGroup` for assertions.